charts/nri-kube-events/templates/configmap.yaml

```diff
@@ -18,3 +18,16 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
 {{ include "nri-kube-events.labels" . | indent 4 }}
+{{ if .Values.config }}
+---
+apiVersion: v1
+kind: ConfigMap
+data:
+    newrelic-infra.yml: |
+{{ toYaml .Values.config | indent 6 }}
```

Comment:
```suggestion
  newrelic-infra.yml: |
{{ toYaml .Values.config | indent 4 }}
```

Reply:
I fixed that, it was coming from the nri-kubernetes chart, but you are right no need for that